mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraDeletedMessageDAO.java

```diff
@@ -123,12 +124,34 @@ public Mono<Void> addDeleted(CassandraId cassandraId, MessageUid uid) {
                 .setLong(UID, uid.asLong()));
     }
 
+    public Mono<Void> addDeleted(CassandraId cassandraId, Flux<MessageUid> uids) {
+        return uids
+            .window(65535)
```

Comment:
just curious, why the number is 65535?

Reply:
But maybe I should reduce this quite a bit here.

Reply:
Yes IMO 1024 was cool on local benches.

Reply:
Then maybe setting up a static constant to make this more clear? :)